react-debugging.html

```diff
@@ -79,17 +79,17 @@ <h1>Draw a Card</h1>
           }
           return(
             <>
-              <button onClick={}>
+              <button onClick={this.drawCard}>
```

Comment:
this.props.drawCard -> drawCard belongs to App.js and is given to Card.js as props.